rdflib/term.py

```diff
@@ -1242,8 +1242,6 @@ def _literal_n3(self, use_plain=False, qname_callback=None):
                     return sub("\\.?0*e", "e", u'%e' % float(self))
                 elif self.datatype == _XSD_DECIMAL:
                     s = '%s' % self
```

Comment:
Can this be reduced to just one line like:
`return '%s' % self`
or even just:
`return str(self)`?

Reply:
Yes, it should have been like that in the first place. Not sure how I missed it. Just made a commit which is being updated in #1058

Reply:
@achaudhary997, PR #1058 seems unrelated. Can you please confirm that's the PR you're referring to here as having addressed this requested update?

Reply:
Yes, that is the same PR. I was told to merge PR #1054, #1058 to master and push the required changes in #1058 
Here is the exact commit link (if required); https://github.com/RDFLib/rdflib/pull/1058/commits/5d07a19d02cde77b38e7fa6303c5dcdd368b805b